### PR TITLE
adding BICAN techniques csv

### PR DESCRIPTION
**Pull request by patrick-lloyd-ray**
@rightbower -- could you please check this work when you get a chance? Thanks!

### CODE REVIEW

File: docs/schemas/BICAN-techniques-simple.csv [hunk 1]
```diff
@@ -0,0 +1,66 @@
+﻿ontology ID,label,alternative term,definition,definition source
+ID,A rdfs:label,A alternative term SPLIT=|,A definition SPLIT=|,A definition source SPLIT=|
+OBI:0003376,droplet-based single-cell RNA sequencing assay,Dropseq,"A sequencing assay that analyzes mRNA transcripts from droplets of individual cells. The droplets are compartmentalized via a microfluidic device. After the cells are compartmentalized, cells in droplets are lysed and the released mRNA hybridizes to the oligo(dT) tract of the primer beads. Then the droplets are pooled and broken to release the beads which are then isolated and reverse-transcribed with template switching. The resulting cDNA strands are PCR-amplified and the sequencing adapters are added. The barcoded mRNA samples are then sequenced.",url:https://doi.org/10.1016/j.cell.2015.05.002
```
Comment (rightbower) [inline docs/schemas/BICAN-techniques-simple.csv]: Illumina and the lab spell this as 'Drop-seq'

File: docs/schemas/BICAN-techniques-simple.csv [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+OBI:0003124,neuron morphology assay,,A 3D neural cell structure determination assay that determines the 3-dimensional configuration of a neuron.,Patrick Ray
+OBI:0003125,mosaicism with repeat frameshift genetic sparse labeling,MORF genetic sparse labeling,A 3D neural cell structure determination assay where sparse and stochastic methods are used to label genetically-defined neurons.,https://doi.org/10.1016/j.neuron.2020.07.019|https://biccn.org/teams/u01-yang
+,serial two-photon tomography assay,STPT,A fluorescence microscopy that generates high-throughput flourescence imaging of mouse brains by integrating two-photon microscopy and tissue sectioning. Serial two-photon tomography generates high-resolution datasets.,url:http://doi.org/10.1038/nmeth.1854
+,mouselight assay,,A serial two photon tomography assay that produces images of mouse brains with sufficient detail to represent axons of individual neurons.,url:http://www.janelia.org/project-team/mouselight/research
```
Comment (rightbower) [inline docs/schemas/BICAN-techniques-simple.csv]: alt-term for this is 'MouseLight' or 'mouselight'

File: docs/schemas/BICAN-techniques-simple.csv [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+OBI:0003105,single-cell combinatorial indexing RNA sequencing assay,sciRNAseq,A single-cell RNA sequencing assay that involves single cell combinatorial indexing ('sci-') strategy to profile the transcriptomes of single cells or nuclei without single cell isolation.,url:https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5894354/
+OBI:0003106,sequential fluorescence in-situ hybridization assay,seqFISH,"A fluorescence in-situ hybridization assay that involves ultra-resolution imaging of cell activity by targetting thousands of single molecules with fluorescent probes which find and hybridize with complementary DNA or RNA sequences in the tissue sample, thus enabling identification of those molecules directly in single cells with their spatial¬†context preserved.",url:https://spatial.caltech.edu/seqfish/
+OBI:0003107,slide-seq assay,,"An RNA-seq assay that involves transfer of RNA from tissue sections onto a surface covered in DNA-barcoded beads with known positions, allowing the locations of the RNA to be inferred by sequencing.",url:https://science.sciencemag.org/content/363/6434/1463
```
Comment (rightbower) [inline docs/schemas/BICAN-techniques-simple.csv]: alt-term is 'Slide-seq'

File: docs/schemas/BICAN-techniques-simple.csv [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+,temp-O sequencing assay,temp-O seq,"A RNA sequencing assay based on BioSpyder Technologies Templated Oligo Detection Assay that targets RNAs with detector olives and removes excess probes and enzymatic inhibitors before the first enzymatic step. Correctly hybridized detector olives are ligated, then amplified through primer landing sites that are share among all probes.",
```
Comment (rightbower) [inline docs/schemas/BICAN-techniques-simple.csv]: alt-term also 'TempO-seq'

File: docs/schemas/BICAN-techniques-simple.csv [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+,x-ray holographic nano-tomography assay,XHN,A 3D structure determination assay that employs X-ray holographic nano-tomography to determine the three-dimensional structure of an entity.,https://doi.org/10.1038/s41593-020-0704-9
+,Pacific Bioscience long-read sequencing assay,,"A DNA sequencing assay wherein a single DNA polymerase enzyme is affixed at the bottom of a zero-mode waveguide (ZMW) with a single molecule of DNA as a template. The Pacific Biotechnologies version is a single-molecule real-time (SMRT) sequencing assay. The ZMW is a structure that creates an illuminated observation volume that is small enough to observe only a single nucleotide of DNA being incorporated by DNA polymerase. Each of the four DNA bases is attached to one of four different fluorescent dyes. When a nucleotide is incorporated by the DNA polymerase, the fluorescent tag is cleaved off and diffuses out of the observation area of the ZMW where its fluorescence is no longer observable. A detector detects the fluorescent signal of the nucleotide incorporation, and the base call is made according to the corresponding fluorescence of the dye.",
```
Comment (rightbower) [inline docs/schemas/BICAN-techniques-simple.csv]: alt-term also 'Pac-Bio long-read sequencing'

File: docs/schemas/BICAN-techniques-simple.csv [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+,enhancer virus labelling assay,enhancer virus labeling,"An imaging assay wherein chromatin accessibility data is collected from cells and enhancers specific to cell types are identified, cloned into recombinant adeno-associated viruses (AAVs) and delivered to the brain, which drive trans gene expression in specific cell types. Then, an image of the marked cell class is generated through fluorescence microscopy.",https://doi.org/10.1016/j.neuron.2021.03.011
+,paired-tag assay,,An ultra-high-throughput histone modification and transcriptome profiling assay that produces cell-type-resolved maps of chromatin state and transcriptome in complex tissues.,https://doi.org/10.1038/s41592-021-01060-3
```
Comment (rightbower) [inline docs/schemas/BICAN-techniques-simple.csv]: alt-term 'Paired-tag'

File: docs/schemas/BICAN-techniques-simple.csv [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+OBI:0002117,whole genome sequencing assay,WGS,A DNA sequencing assay that intends to provide information about the sequence of an entire genome of an organism.,PMID:25827230|PMID:23095910
+OBI:0002985,magnetic resonance imaging assay,MRI,An imaging assay in which nuclear magnetic resonance is used to produce information about the interior structure and composition of an input material entity.,url:https://en.wikipedia.org/wiki/Magnetic_resonance_imaging
+OBI:0000626,DNA sequencing assay,nucleotide sequencing,A sequencing assay which determines information on the sequence of a DNA molecule.,url:https://en.wikipedia.org/wiki/DNA_sequencing
```
Comment (rightbower) [inline docs/schemas/BICAN-techniques-simple.csv]: alt-term 'DNAseq'

File: docs/schemas/BICAN-techniques-simple.csv [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+,nanopore DNA sequencing assay,,"A DNA sequencing assay that provides direct, real-time analysis of long DNA or RNA fragments by monitoring changes to an electrical current as nucleic acids are passed through a protein nanopore. The resulting signal is decoded to provide the specific DNA or RNA sequence.",https://nanoporetech.com/applications/dna-nanopore-sequencing
+,single cell isoform RNA sequencing assay,ScISOr-seq,An isoform sequencing assay that begins with bulk tissue and identifies single-cell types and their full-length RNA isoforms without fluorescence-activated cell sorting.,https://doi.org/10.1038/nbt.4259
+,single nucleus methyl-3C sequencing assay,sn-m3c-seq,A single-cell RNA sequencing assay that jointly profiles chromatin conformation and DNA methylation information from the same cell.,https://doi.org/10.1038/s41592-019-0547-z
+,single nucleus methyl-C sequencing assay version 2,snmC-seq2,"A library preparation method for single-nucleus methylome library preparation that provides significant improvements in read mapping, amount of artifactual reads, throughput, library complexity, and coverage uniformity over snmC-seq.",
+,mC-seq3,,,
```
Comment (rightbower) [inline docs/schemas/BICAN-techniques-simple.csv:55]: alt-term 'mC-seq3'

**Review by rightbower — COMMENTED** [review]
added some terms from our CV for those missing alternative terms or spellings?

File: docs/schemas/BICAN-techniques-simple.csv (hunk 1 above)
Comment (patrick-lloyd-ray) [inline docs/schemas/BICAN-techniques-simple.csv]:
```suggestion
OBI:0003376,droplet-based single-cell RNA sequencing assay,Dropseq|Drop-seq,"A sequencing assay that analyzes mRNA transcripts from droplets of individual cells. The droplets are compartmentalized via a microfluidic device. After the cells are compartmentalized, cells in droplets are lysed and the released mRNA hybridizes to the oligo(dT) tract of the primer beads. Then the droplets are pooled and broken to release the beads which are then isolated and reverse-transcribed with template switching. The resulting cDNA strands are PCR-amplified and the sequencing adapters are added. The barcoded mRNA samples are then sequenced.",url:https://doi.org/10.1016/j.cell.2015.05.002

```

File: docs/schemas/BICAN-techniques-simple.csv (hunk 2 above)
Comment (patrick-lloyd-ray) [inline docs/schemas/BICAN-techniques-simple.csv]:
```suggestion
,mouselight assay,MouseLight|mouselight,A serial two photon tomography assay that produces images of mouse brains with sufficient detail to represent axons of individual neurons.,url:http://www.janelia.org/project-team/mouselight/research

```

File: docs/schemas/BICAN-techniques-simple.csv (hunk 3 above)
Comment (patrick-lloyd-ray) [inline docs/schemas/BICAN-techniques-simple.csv]:
```suggestion
OBI:0003107,slide-seq assay,Slide-seq,"An RNA-seq assay that involves transfer of RNA from tissue sections onto a surface covered in DNA-barcoded beads with known positions, allowing the locations of the RNA to be inferred by sequencing.",url:https://science.sciencemag.org/content/363/6434/1463

```

File: docs/schemas/BICAN-techniques-simple.csv (hunk 4 above)
Comment (patrick-lloyd-ray) [inline docs/schemas/BICAN-techniques-simple.csv]:
```suggestion
,temp-O sequencing assay,temp-O seq|TempO-seq,"A RNA sequencing assay based on BioSpyder Technologies Templated Oligo Detection Assay that targets RNAs with detector olives and removes excess probes and enzymatic inhibitors before the first enzymatic step. Correctly hybridized detector olives are ligated, then amplified through primer landing sites that are share among all probes.",

```

File: docs/schemas/BICAN-techniques-simple.csv (hunk 5 above)
Comment (patrick-lloyd-ray) [inline docs/schemas/BICAN-techniques-simple.csv]:
```suggestion
,Pacific Bioscience long-read sequencing assay,Pac-Bio long-read sequencing,"A DNA sequencing assay wherein a single DNA polymerase enzyme is affixed at the bottom of a zero-mode waveguide (ZMW) with a single molecule of DNA as a template. The Pacific Biotechnologies version is a single-molecule real-time (SMRT) sequencing assay. The ZMW is a structure that creates an illuminated observation volume that is small enough to observe only a single nucleotide of DNA being incorporated by DNA polymerase. Each of the four DNA bases is attached to one of four different fluorescent dyes. When a nucleotide is incorporated by the DNA polymerase, the fluorescent tag is cleaved off and diffuses out of the observation area of the ZMW where its fluorescence is no longer observable. A detector detects the fluorescent signal of the nucleotide incorporation, and the base call is made according to the corresponding fluorescence of the dye.",

```

File: docs/schemas/BICAN-techniques-simple.csv (hunk 6 above)
Comment (patrick-lloyd-ray) [inline docs/schemas/BICAN-techniques-simple.csv]:
```suggestion
,paired-tag assay,Paired-tag,An ultra-high-throughput histone modification and transcriptome profiling assay that produces cell-type-resolved maps of chromatin state and transcriptome in complex tissues.,https://doi.org/10.1038/s41592-021-01060-3

```

File: docs/schemas/BICAN-techniques-simple.csv (hunk 7 above)
Comment (patrick-lloyd-ray) [inline docs/schemas/BICAN-techniques-simple.csv]:
```suggestion
OBI:0000626,DNA sequencing assay,nucleotide sequencing|DNAseq,A sequencing assay which determines information on the sequence of a DNA molecule.,url:https://en.wikipedia.org/wiki/DNA_sequencing
```

**Pull request merged**